open-craft/taxonomy-connector
# [BB-6856] Add new management command for skill tags verification

## Description
This PR adds a new management command `finalize_xblockskill_tags` that checks the votes on tags and marks them as verified. This command will have to be run periodically - can be configured as a cron job.

## Supporting information

- OpenCraft internal task: [BB-6856](https://tasks.opencraft.com/browse/BB-6856)

## Testing Instructions
1. Install this branch in Course Discovery.
2. Ensure the host platform contains the settings values for `MIN_VOTES_FOR_SKILLS` and `RATIO_THRESHOLD_FOR_SKILLS`.
3. From within the discovery django shell, run the management command:
```
./manage.py finalize_xblockskill_tags
```
4. If the tags have the required number of votes, it should be marked as verified.
